docs/config.md

```diff
@@ -1800,35 +1802,40 @@ Type: `duration`
 
 ### `Swarm.ResourceMgr`
 
-**EXPERIMENTAL: `Swarm.ResourceMgr` configuration will change in future release**
-
 The [libp2p Network Resource Manager](https://github.com/libp2p/go-libp2p-resource-manager#readme) allows setting limits per a scope,
 and tracking recource usage over time.
 
 #### `Swarm.ResourceMgr.Enabled`
 
-**EXPERIMENTAL: `Swarm.ResourceMgr` is in active development, enable it only if you want to provide maintainers with feedback**
-
-
 Enables the libp2p Network Resource Manager and auguments the default limits
 using user-defined ones in `Swarm.ResourceMgr.Limits` (if present).
 
 Various `*rcmgr_*` metrics can be accessed as the prometheus endpoint at `{Addresses.API}/debug/metrics/prometheus` (default: `http://127.0.0.1:5001/debug/metrics/prometheus`)
 
-Default: `false`
+Default: `true`
 
 Type: `flag`
 
-#### `Swarm.ResourceMgr.Limits`
+#### `Swarm.ResourceMgr.MaxMemory`
 
-**EXPERIMENTAL: `Swarm.ResourceMgr.Limits` configuration will change in future release, exposed here only for convenience**
+Define the maximum amount of memory used by libp2p.
 
-Map of resource limits [per scope](https://github.com/libp2p/go-libp2p-resource-manager#resource-scopes).
+Default: `[TOTAL_MEMORY]/8`
+Type: `integer64`
+
+#### `Swarm.ResourceMgr.MaxFileDescriptors`
+
+Define the maximum amount of file descriptors used by libp2p.
+
+Default `[TOTAL_FILE_DESCRIPTORS]/2`
```

Comment:
```suggestion
Default `[TOTAL_SYSTEM_FILE_DESCRIPTORS]/2`
```

Should we add a comment on what happens in Windows cases?  I assume we'll follow https://github.com/libp2p/go-libp2p/search?q=getNumFDs and set it to infinite by default (or do we want to set to a different value).